packages/kit/src/runtime/server/page/render.js

```diff
@@ -169,6 +169,10 @@ export async function render_response({
 		headers['cache-control'] = `${is_private ? 'private' : 'public'}, max-age=${maxage}`;
 	}
 
+	if (!options.floc) {
+		headers['permissions-policy'] = 'interest-cohort=()';
```

Comment:
Are we worried about this being unsafe? The example PHP code in the Wordpress blog post about this had a bit more logic in place for dealing with an existing `permissions-policy` header.

Reply:
We don't currently expose a mechanism for returning custom headers from pages, so this is safe — the only caveat being that user code in `handle` needs to be aware of existing headers

Reply:
If someone returned a `permissions-policy` header from an _endpoint_ though, this would clobber it, right? Are you concerned about that?

Reply:
This doesn't touch endpoint headers at all, only pages (AFAICT FLoC is only relevant there?)

Reply:
Oh okay. Well I suppose someone could still be returning HTML from an endpoint that gets navigated to at some point, but that's probably an edge case. Adding this header to all page responses by default sounds reasonable to me.